model-optimizer/mo/utils/custom_replacement_config.py

```diff
@@ -345,6 +346,7 @@ def parse_custom_replacement_config_file(file_name: str):
     try:
         with open(file_name, 'r') as f:
             data = json.load(f)
```

Comment:
please move this piece of code into `validate_json_config` function and call this function as `load_and_validate_json_config`